docs/data/charts/line-demo/line-demo.md

```diff
@@ -31,3 +31,15 @@ components: LineChart, LineElement, LineHighlightElement, LineHighlightPlot, Lin
 ## LineChartConnectNulls
 
 {{"demo": "LineChartConnectNulls.js"}}
+
+## Line with predicition
+
+To show that some part of the data have a different status, you can vary the rendering.
```

Comment:
What do you mean by "vary the rendering"?

Reply:
Triky to explain. The idea is to get multiple styles (in the example a line with dashed and non dashed area) on the same line